### DISCUSSION TITLE
For field tooltip in VSCode please add support of UTF-8 icons and display field label too.

### DISCUSSION THREAD

**Discussion by PawelWozniak** [forcedotcom/salesforcedx-vscode#5369] [Ideas]
**Is your feature request related to a problem? Please describe.**
I wanted to write a question to the user about some fields that I saw in the code. But in VSCode I see only API names and for the user I need to have labels. Currently I need to go to SF setup find fields and copy over labels which is time consuming.

**Describe the solution you'd like**
Since some fields have different labels than API names I would like to see field label in VSCode popup that is shown when mouse is over the field API name.

Now I see data type, object and api name --> String Contract.Status 
Would like to see that + label --> String Contract.Status (Document Status)
I mean this popup:
![image](https://github.com/forcedotcom/salesforcedx-vscode/assets/5740564/dde0ab49-1355-45f6-a606-f2a866e543f2)

Additionally I would like to see UTF-8 icons used there, in this case 👉 symbols should be there:
<img width="317" alt="2024-01-19_10h32_54" src="https://github.com/forcedotcom/salesforcedx-vscode/assets/5740564/464c92af-7ed8-4e3c-97a2-61e8465f6791">
SF is not respecting new lines in tooltip that is the reason why I am using hands to point new bullet.

**Describe alternatives you've considered**
Not considered as it seems to be simplest solution.

**Comment by git2gus** [comment]
This issue has been linked to a new work item: W-14853133

**Comment by peternhale** [comment]
@PawelWozniak Thank you for the feature request.
